emcc.py

```diff
@@ -1520,9 +1521,9 @@ def compile_source_file(i, input_file):
             # Sort arg tuples and pass the extracted values to link.
             shared.Building.link(linker_inputs, specified_target)
         logging.debug('stopping at bitcode')
-        if final_suffix.lower() in ['so', 'dylib', 'dll']:
+        if final_suffix.lower() in ['.so', '.dylib', '.dll']:
           logging.warning('Dynamic libraries (.so, .dylib, .dll) are currently not supported by Emscripten. For build system emulation purposes, Emscripten'
-            + ' will now generate a static library file (.bc) with the suffix \'.' + final_suffix + '\'. For best practices,'
+            + ' will now generate a static library file (.bc) with the suffix ' + final_suffix + '. For best practices,'
```

Comment:
Could probably keep the `'`s around the suffix, `'... the suffix \'' + final_suffix + '\'. For best...'`

Nitpicking the line that was already here (don't need to do these): could also change the `'`s into `"`s and not need to escape the `'`s as `\'`. Could also use string formatting to make it clearer how `final_suffix` is presented.